op-e2e/actions/l2_batcher.go

```diff
@@ -231,6 +248,13 @@ func (s *L2Batcher) ActL2BatchSubmit(t Testing, txOpts ...func(tx *types.Dynamic
 		t.Fatalf("failed to output channel data to frame: %v", err)
 	}
 
+	payload := data.Bytes()
+	if s.l2BatcherCfg.UsePlasma {
+		var err error
+		payload, err = s.l2BatcherCfg.PlasmaDA.SetInput(context.TODO(), payload)
```

Comment:
```suggestion
		payload, err = s.l2BatcherCfg.PlasmaDA.SetInput(t.Ctx(), payload)
```
We should have contexts available, try to avoid `context.TODO()`.

Reply:
Removed